src/output_writers.jl

```diff
@@ -100,9 +105,9 @@ noinit(args...) = nothing
                      prefix="", init=noinit, including=[:grid, :eos, :constants, :closure],
                      part=1, max_filesize=Inf, force=false, async=false, verbose=false)
 
-Construct an `OutputWriter` that writes `label, func` pairs in the dictionary `outputs` to
-a single JLD2 file, where `label` is a symbol that labels the output and `func` is a
-function of the form `func(model)` that returns the data to be saved.
+Construct a `JLD2OutputWriter` that writes `label, func` pairs in the dictionary `outputs`
```

Comment:
`outputs` can be either a `NamedTuple` or a `Dictionary`; probably this should be mentioned. An example may be apropos...